src/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,15 @@
+import { UserIcon } from '../icons';
+
+import styles from './avatar.module.scss';
+
+export interface AvatarProps {
+    imageSrc: string | undefined;
+}
+
+export const Avatar = ({ imageSrc }: AvatarProps) => {
+    return (
+        <div className={styles.root}>
```

Comment:
The root is redundant, but ok.

```tsx
<img className={styles.root} src={imageSrc} alt="">
```

```css
img.root { ... }
svg.root { ... }
```

Reply:
maybe redundant, but I'm sleeping better knowing that my react components doesn't change their root element (not sure why)